src/PrestaShopBundle/Controller/Admin/Improve/Design/PositionsController.php

```diff
@@ -157,7 +157,7 @@ public function unhookAction(Request $request)
             $module = $this->get('prestashop.adapter.legacy.module')->getInstanceById($moduleId);
             $hook = new Hook($hookId);
 
-            if (!$validateAdapter->isLoadedObject($module)) {
+            if (!$module) {
```

Comment:
I guess this is to avoid an error when null is returned, but is it really ok to remove the validation? couldn't it be something like
`if (!$module || !$validateAdapter->isLoadedObject($module))`

Reply:
isLoadedObject try to check if $module have an id. Which is not happened everytime (don't know why). I try with many values and if a module isn't found it return false.

Reply:
fair enough